api/operate/operate-api-introduction.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Operate API Explorer
+description: "Operate API is a REST API and provides searching, getting, and changing Operate data. Requests and responses are in JSON."
+---
+
+## Introduction
+
+Operate API is a REST API and provides searching, getting, and changing Operate data.
+Requests and responses are in JSON notation. Some objects have additional endpoints.
```

Comment:
JSON stands for `JavaScript Object Notation` so maybe we can remove the extra `notation` word.

Reply:
I forgot to close this PR when I opened a replacement! In #2944, this file is no longer moved to this new section of the docs. 

The language here already exists in the docs, and isn't a core part of this work. I welcome a separate PR to change it in its source, but this is out of scope for this work.